.github/workflows/integration-build-product-composite.yml

```
@@ -1815,6 +1774,10 @@ jobs:
   publish:
     needs: [Install-Robot, Install-Heterogeneous-Fleet-Single-VM,Install-Heterogeneous-Fleet-Multi-VM, Install-Simulator-Robot]
```

In order for this to run even if Install-Simulator-Robot is skipped you need the following condition:

```suggestion
    needs: [Install-Robot, Install-Heterogeneous-Fleet-Single-VM,Install-Heterogeneous-Fleet-Multi-VM, Install-Simulator-Robot]
    if: ${{ always() && (needs.Install-Simulator-Robot.result == 'success' ||  needs.Install-Simulator-Robot.result == 'skipped'  ) }}
```